aya/src/programs/perf_event.rs

```diff
@@ -50,30 +50,44 @@ pub enum SamplePolicy {
     Frequency(u64),
 }
 
-/// The scope of a PerfEvent
+/// A flag whose bits indicate the fields to include in the event samples.
+#[derive(Debug, Clone)]
+pub struct SampleType(u64);
+
+/// "Wake up" overflow notification policy.
+/// Overflows are generated only by sampling events.
```

Comment:
Missing newline between the two lines. The format is:

[summary]
[newline]
[paragraphs separated by newlines]